inspire/modules/authors/recordext/fields/author.cfg

```diff
@@ -0,0 +1,438 @@
+                    'numeration': {'type': 'string', 'nullable': True,
+                                   'regex': __import__('flask').current_app.config['AUTHORS_NAME_NUMERATIONS']},
+                    'title': {'type': 'string', 'nullable': True,
+                              'allowed': __import__('flask').current_app.config['AUTHORS_NAME_TITLES']},
+                    'status': {'type': 'string', 'nullable': True,
+                               'allowed': __import__('flask').current_app.config['AUTHORS_NAME_STATUSES']},
+                    'preferred_name': {'type': 'string', 'nullable': True,
+                                       'empty': False}}}}
+    creator:
+        marc, "100__", {'last': util_split(value['a'], ',', 0),
+                        'first': util_split(value['a'], ',', 1),
+                        'numeration': value['b'],
+                        'title': value['c'],
+                        'status': value['g'],
+                        'preferred_name': value['q']}
+    producer:
+        json_for_marc(), {"100__a": "last + ', ' + first",
+                          "100__b": "numeration", "100__c": "title",
+                          "100__g": "status", "100__q": "preferred_name"}
+
+other_names:
+    """Contains other variation of names.
+
+    Usually a different form of writing the primary name.
+
+    TO DO: define a type inside list
+    """
+    schema:
+        {'other_names': {'type': 'list', 'force': True}}
+    creator:
+        marc, "400__", value['a']
+    producer:
+        json_for_marc(), {"400__", ""}
+
+previous_names:
+    """Contains a previous name.
+
+    For example, it can be a maiden's name.
+
+    TO DO: define a type inside list
+    """
+    schema:
+        {'previous_name': {'type': 'list', 'force': True}}
+    producer:
+        json_for_marc(), {"400__a": "", "400__w": "a"}
+
+native_name:
+    """Stores name in native form."""
+    schema:
+        {'native_name': {'type': 'string'}}
+    creator:
+        marc, "880__", value['a']
+    producer:
+        json_for_marc(), {"880__": ""}
+
+dates:
+    """Stores birth and death dates.
+
+    Example:
+    +---------+---------------------+
+    | 100__d  | 1943-2001           |
+    +---------+---------------------+
+    """
+    schema:
+        {'dates': {'type': 'dict',
+         'schema': {'birth': {'type': 'integer', 'nullable': True,
+                              'min': __import__('flask').current_app.config['INSPIRE_YEAR_MIN'],
+                              'max': __import__('flask').current_app.config['INSPIRE_YEAR_MAX']
+                              },
+                    'death': {'type': 'integer', 'nullable': True,
+                              'min': __import__('flask').current_app.config['INSPIRE_YEAR_MIN'],
+                              'max': __import__('flask').current_app.config['INSPIRE_YEAR_MAX']}}}}
+    creator:
+        @only_if_master_value(value["d"])
```

Comment:
Hopefully it would be only a temporary workaround. The problem is that if the `100__` field doesn't contain a correct `date`, the JSONAlchemy returns a `KeyError`. This decorator requires the value. However, it doesn't check if it is correct, that's why the test is skipped.

I could write a function to check if the dates are in correct format, but it should be handled by JSONAlchemy. I'm waiting for an answer from Esteban.